### ISSUE TITLE
Datepicker: Build redesign for Year/Month Picker

### ISSUE THREAD

**Issue by clepore** [infor-design/enterprise#1958]
**Is your feature request related to a problem? Please describe.**
There was a request to redesign parts of the datepicker component some details explained on the reasoning on 

**Describe the solution you'd like**
Build the provided design assets from https://github.com/infor-design/design-system/issues/352 once it is finished.

**Design**
https://www.dropbox.com/s/x6vgy3e9650miv7/IDS_Datepicker_02.sketch?dl=0 be sure to download the file to see all layers in the design the dropdox view just shows the main layer.

**Implementation Notes**
- http://latest-enterprise.demo.design.infor.com/components/datepicker/example-month-year-format.html should be updated so that the current date and month picker is used and all examples on this page should be adapted.
- http://latest-enterprise.demo.design.infor.com/components/datepicker/example-month-year-picker.html should also be changed probably so that clicking the year/month shows the view on the design?

**Comment by janahintal** [comment]
Hi @tmcconechy , Noticed that in https://4200-beta1-enterprise.demo.design.infor.com/components/datepicker/example-index.html Month Year picker doesnt have select button. Thus, the user has to click it twice to select both desired Month/year.

**Comment by tmcconechy** [comment]
Hi @janahintal 
I see what you mean. However, this was the design. You have to open it twice to select both month and year. This is possibly debatable. Can you make a new issue for that and we can review and discuss if it needs a change for this?

**Comment by janahintal** [comment]
Noted on that @tmcconechy . Created #2557 & will move this to Done. Thanks! :)
